test/unit/dialects/postgres/query-generator.test.js

```diff
@@ -1331,5 +1331,53 @@ if (dialect.startsWith('postgres')) {
         });
       });
     });
+
+    describe('Custom Schema Functions', () => {
+      beforeEach(function () {
+        this.queryGenerator = new QueryGenerator({
+          sequelize: this.sequelize,
+          _dialect: this.sequelize.dialect,
+        });
+        // Add custom schema to sequelize object
+        this.queryGenerator.sequelize.options.schema = 'custom';
+      });
+      afterEach(function () {
+        // Remove custom schema in order to not interfere with other tests
+        this.queryGenerator.sequelize.options.schema = null;
+      });
+
+      const customSchemaSuites = {
+        showTablesQuery: [
+          {
+            title: 'showTablesQuery should use correct custom schema',
```

Comment:
```suggestion
            title: 'showTablesQuery defaults to the schema set in Sequelize options',
```

Same for the other 2

Reply:
fixed